components/countryInfo.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const CountryInfo = () => {
+  return (
+    <div className='text-white md:py-10 py-20'>
```

Comment:
- [ ] [Optional] Fantastic work! I truly appreciate how you implemented the map using the `iframe`. However, it would be even more polished if you wrapped your entire component code with a `section` tag. Keep up the great work!